### PR TITLE
fix: add analytics script

### PR DESCRIPTION
**Pull request by lucasgdev**
## What this PR is changing?

It's including an analytics snippet.

### CODE REVIEW

File: web/src/pages/index.js [hunk 1]
```diff
@@ -1,16 +1,32 @@
...
   const themeToUse = lightTheme;
 
   return (
-    <ThemeProvider theme={themeToUse}>
-      <App />
-    </ThemeProvider>
+    <>
+      <Helmet>
+        <script async defer src="https://sa.moul.io/latest.js"></script>
+        <noscript>
```
Comment (moul) [inline web/src/pages/index.js:15]: the `<noscript>` tag couldn't be located in the `<head>`

File: web/gatsby-config.js [hunk 2]
```diff
@@ -66,5 +66,14 @@ module.exports = {
         trackingId: "UA-47629346-5",
```
Comment (lucasgdev) [inline web/gatsby-config.js:66]: @moul we have this GA plugin and config, we should keep this and replicate on the www?

**Comment by lucasgdev** [comment]
@moul ♻️

File: web/gatsby-config.js (hunk 2 above)
Comment (moul) [reply web/gatsby-config.js:66]: Keep it for now, we’ll remove it in few weeks

No need to replicate on www

**Pull request merged**